Sources/Core/AST/Stmt/CondCompilationStmt.swift

```diff
@@ -0,0 +1,82 @@
+/// A conditional-compilation statement.
+public struct CondCompilationStmt: Stmt {
```

Comment:
I've been trying to eliminate most abbreviations from the code.
```suggestion
public struct ConditionalCompilationStmt: Stmt {
```
or perhaps even better:
```suggestion
public struct ConditionallyCompiledStmt: Stmt {
```